ripe/atlas/tools/commands/probes.py

```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```

Comment:
Though I suppose you could support comma-separated values for `--status` and that'd be totally ok.  No need to implement `--status` *and* `--statuses`.